features/attach_validtoken.feature

```diff
@@ -494,3 +494,73 @@ Feature: Command behaviour when attaching a machine to an Ubuntu Advantage
           | xenial  | disabled |
           | bionic  | disabled |
           | focal   | n/a      |
+
+    @series.all
+    @uses.config.machine_type.lxd.container
+    Scenario Outline: Attach and Check for contract change in status checking
+       Given a `<release>` machine with ubuntu-advantage-tools installed
+       When I attach `contract_token` with sudo
+       Then stdout matches regexp:
+       """
+       UA Infra: ESM enabled
+       """
+       And stdout matches regexp:
+       """
+       This machine is now attached to
+       """
+       And stdout matches regexp:
+       """
+       SERVICE       ENTITLED  STATUS    DESCRIPTION
```

Comment:
This will need a `+` as soon as realtime is merged
```suggestion
       SERVICE       +ENTITLED  STATUS    DESCRIPTION
```

Reply:
It will only affect ENTITLED and not the other fields after?